docs/source/managed-federation/uplink.mdx

```diff
@@ -56,6 +61,21 @@ By default, the gateway retries a single poll request a number of times equal to
 
 > Even if a particular poll request fails _all_ of its retries, the gateway continues polling as usual at the next interval (with its own set of retries if needed). In the meantime, the gateway continues using its most recently obtained configuration.
 
+### Poll interval
+
+You can configure the interval at which your gateway polls Apollo Uplink like so:
+
+```js
+const { ApolloGateway } = require('@apollo/gateway');
+
+// ...
+
+const gateway = new ApolloGateway({
+  experimental_pollInterval: 15000 // 15 seconds
```

Comment:
I have a PR open (should be landing soon) which deprecates this in favor of `pollIntervalInMs`. Can this PR wait for that change to land? :)

Reply:
Also you _can_ just target my branch and merge this into my PR if you'd like! It'll conveniently get ported to 2.x as well.

`trevor/improve-supergraphSdl`

Reply:
Ah yeah sure! Will edit and retarget.